docs/meshcloud.policies.md

```diff
@@ -5,11 +5,51 @@ title: meshPolicies
 
 ## Introduction: what are meshPolicies?
 
-A meshPolicy describes a restriction between two [meshObjects](meshcloud.index.md#introduction) that are globally defined by your organization. Each meshPolicy describes which [tag](meshcloud.metadata-tags.md) values have to be present on both meshObjects, in order to comply with the meshPolicy. meshPolicies are enforced in various places when attaching meshObjects. How meshPolicies can be configured is described [here](administration.mesh-policies.md).
+A meshPolicy describes a restriction between two meshSubjects. meshSubjects are [meshObjects](meshcloud.index.md#introduction) that are globally defined by your organization but not the other way around. This means only selected meshObjects are meshSubjects.
+
+> **meshCustomer**, **meshProject**, **meshUser/Group** and **meshLandingZone** are meshSubjects
+
+You can find the authoritative meshSubject on the left side and the affected meshSubject on the right side. Each meshPolicy describes which [tag](meshcloud.metadata-tags.md) values have to be present on both meshSubjects, in order to comply with the meshPolicy. To decide if a meshPolicy is compliant it is necessary to select an evaluation strategy how you want to evaluate the authoritative meshSubject against the affected meshSubject. meshPolicies are enforced in various places when attaching meshSubjects. How meshPolicies can be configured is described [here](administration.mesh-policies.md).
+
+> meshPolicies will only be enforced for meshCustomers. meshPartners are excluded.
+
+### Authoritative and affected meshSubject
```

Comment:
d: I'm not sure about the terms authoritative meshSubject and affected meshSubject since we never mentioned it in the panel. But for comprehension I can understand why it can be useful. 
But on the other hand we only have two authoritative meshSubject saying meshCustomer and meshProject. So that another possibility could be to use them right away.  Saying not calling the section "Authoritative and affected meshSubject" but having for example this section:

**What can meshPolicies enforce?**

meshPolicies can enforce that
- meshCustomer restricts the meshProjects 
- meshCustomer restricts new assignments meshUsers/Groups
- meshProject restricts new assignments meshUsers/Groups
- meshProject restricts new assignments meshLandingZones 

based on the defined tag in the meshPolicies.

So just an idea. Also fine "authoritative meshSubject and affected meshSubject" section :)

Reply:
I prefer the direct naming because in the future we can have other auth meshSubjects besides the meshCustomer and meshProject.